svc/FetchService.js

```diff
@@ -4,14 +4,14 @@
  *
  * Copyright © 2021 Extremely Heavy Industries Inc.
  */
-import {HoistService, XH} from '@xh/hoist/core';
+import {HoistService, XH, AppState} from '@xh/hoist/core';
 import {Exception} from '@xh/hoist/exception';
-import {isLocalDate} from '@xh/hoist/utils/datetime';
+import {isLocalDate, SECONDS, ONE_MINUTE, olderThan} from '@xh/hoist/utils/datetime';
 import {throwIf} from '@xh/hoist/utils/js';
 import {StatusCodes} from 'http-status-codes';
 import {isDate, isFunction, isNil, omitBy} from 'lodash';
 import {stringify} from 'qs';
-import {SECONDS} from '@xh/hoist/utils/datetime';
+import {never} from '@xh/hoist/promise';
```

Comment:
For the real suggestion below (my use of fancy suggestion feature a bit sub-optimal here...)

```suggestion
import {never, wait} from '@xh/hoist/promise';
```